doc/doap-iframe.html

```diff
@@ -0,0 +1,12 @@
+<html>
+	<head>
+		<script>
+			function resizeIframe(obj) {
+				obj.style.height = obj.contentWindow.document.documentElement.scrollHeight + 20 + 'px';
+			}
+		</script>
```

Comment:
please use css for styling

Reply:
Unfortunately, that is not possible. I added a corresponding comment explaining it to that function.